src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,25 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.security.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector inject = Injector.getInstance("mate.academy");
+
+    public static void main(String[] args) throws RegistrationException {
```

Comment:
I think its better to use try catch instead of throws...

Reply:
idk, no reason what use here l guess, last my work was approved with that realization

Reply:
Usually yes, but this is a testing method so if you don't need to process this exception let's leave it like this